apps/condo/domains/ticket/schema/Ticket.test.js

```diff
@@ -1159,6 +1157,14 @@ describe('Ticket', () => {
             })
         })
 
+        // TODO: DOMA-10832 add check employee organization in Ticket access
+        test.skip('user: cannot create ticket executor field with user from different organization', async () => {
+            const client = await makeClientWithProperty()
+            const client2 = await makeClientWithProperty()
+
+            await createTestTicket(client, client.organization, client.property, { executor: { connect: { id: client2.user.id } } })
```

Comment:
An error should be expected here. Probably in another pr

Reply:
And now everything will work correctly for us(